icu4c/source/common/ucurr.cpp

```diff
@@ -2295,7 +2234,7 @@ ucurr_countCurrencies(const char* locale,
 {
     int32_t currCount = 0;
 
-    if (ec != nullptr && U_SUCCESS(*ec)) 
+    if (ec != nullptr && U_SUCCESS(*ec))
```

Comment:
Unrelated formatting change.

Reply:
I think we should remove those junk whenever we see it. I do not mind to revert the { change, but this is an extra space or tab.